app/services/database_schema_service.py

```diff
@@ -44,6 +44,15 @@ async def _get(
         )
         return result.scalars().first()
 
+    async def _get_by_fully_qualified_name(self, fully_qualified_name: str) -> typing.Optional[DatabaseSchema]:
```

Comment:
I'm thinking about refactoring the services and create an "AssetBaseService", with at least the find by fqn. The code is the same, the only thing that changes is the target class and loading options. But this could be in future.